etc/restic/_global.env

```diff
@@ -0,0 +1,16 @@
+# B2 credentials.
+# Extracted settings so both systemd timers and user can just source this when want to work on my B2 backup.
+# See https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html
+
+# these variables are sourced FIRST, and any values inside of *.env files for
+# specific configurations will over-ride if also defined there
```

Comment:
... thus add here as first line

```
source /etc/restic/_global.env
```

Reply:
don't need to; the .system file grabs it:

```
Description=Backup with restic to Backblaze B2
OnFailure=status-email-user@%n.service

[Service]
Type=simple
Nice=10
ExecStart=/usr/local/sbin/restic_backup.sh
EnvironmentFile=-/etc/restic/_global.env
EnvironmentFile=/etc/restic/%I.env
```

Reply:
I know. The intention is to make it self-contained:

```bash
$ . /etc/restic/example.env
$ restic snapshots
```

vs

```bash
$ . /etc/restic/_global.env
$ . /etc/restic/example.env
$ restic snapshots
```